exec/views.py

```diff
@@ -17,6 +19,8 @@ class Index(ListView):
     def get_context_data(self, *args, **kwargs):
         ctxt = super().get_context_data(*args, **kwargs)
         ctxt['resp'] = 'full' in self.request.GET
+        history, d = TextProperty.objects.get_or_create(key='exec_history', defaults="")
```

Comment:
There is an easier way to do this in future. In the template, `{% load property_tags %}` and then  `{% text_setting "exec_history" %}` (or `{% text_setting "exec_history" as history %}{{ history|parse_md_safe }}`) where you want the content

Reply:
resolved via c1ec36c